game_board.py

```diff
@@ -148,60 +148,120 @@ def which_color_pair(stone_marker):
         color = 3
     return color
 
-def move_down(plus_list,current_row):
-    """ Finds the "+" char that is below the current_row. If the current "+" char is at the bottom, 
-        it finds the "+" char at the top with the same x position
-    Returns the new y position of the "+" 
+def move_down(plus_list,loc):
```

Comment:
`loc` refers to the current location right? I would suggest renaming it to `current_location`